scripts/shell.sh

```diff
@@ -4,4 +4,4 @@ set -e
 cd $(dirname $0)/..
 . ./scripts/dapper-common
 
-exec dapper -s
+exec dapper -d -s
```

Comment:
Dapper is not too verbose with debug. And you probably want to know the arguments to docker if you're using ./scripts/shell.sh

I've added this to the PR because I found myself adding this temporarily a few times already.